app/models/comment.rb

```diff
@@ -1,2 +1,8 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  after_save :update_comment_count
+  def update_comment_count
+    post.upadate(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_comment_count` should be marked as private after implementing the `after_save` method.